v3/src/components/graph/utilities/graph_utils.ts

```diff
@@ -95,21 +87,22 @@ export function getPointTipText(dataset: IDataSet, caseID: string, attributeIDs:
         isNumeric = attribute?.type === 'numeric',
         value = isNumeric ? float(dataset?.getNumeric(caseID, attrID) ?? 0) :
           dataset?.getValue(caseID, attrID)
-      return `${name}: ${value}`
+      return (value && (isNumeric && isFinite(value)) || (!isNumeric && value !== '')) ? `${name}: ${value}` : ''
     }))
   // Caption attribute can also be one of the plotted attributes, so we remove dups and join into html string
-  return Array.from(new Set(attrArray)).join('<br>')
+  return Array.from(new Set(attrArray)).filter(anEntry => anEntry !== '').join('<br>')
```

Comment:
Assuming `anEntry` can't be another falsy value, could also use:
```suggestion
  return Array.from(new Set(attrArray)).filter(anEntry => !!anEntry).join('<br>')
```

Reply:
anEntry _will_ be a string, but if it's an _empty_ string we don't want to include it.

Reply:
Exactly -- my proposed change should have the same effect with less code.